sample_config/database-layout.sql

```diff
@@ -49,7 +58,52 @@ ALTER TABLE public.timed_thinclient_to_vm_mapping OWNER TO "vdi-dbadmin";
+SELECT f.id,
+       f.thinclient,
+       f.vm,
+       f.resolution,
+       f.start_date,
+       f.end_date,
+       f.prio,
+       f.shutdown_vm
+FROM
+  (
+     (SELECT timed_thinclient_to_vm_mapping.id,
+             timed_thinclient_to_vm_mapping.thinclient,
+             timed_thinclient_to_vm_mapping.vm,
+             timed_thinclient_to_vm_mapping.resolution,
+             timed_thinclient_to_vm_mapping.start_date,
+             timed_thinclient_to_vm_mapping.end_date,
+             0 AS prio,
+             timed_thinclient_to_vm_mapping.shutdown_vm
+      FROM timed_thinclient_to_vm_mapping
+      WHERE ((timed_thinclient_to_vm_mapping.start_date IS NULL)
+             AND (timed_thinclient_to_vm_mapping.end_date IS NULL))
+      UNION SELECT timed_thinclient_to_vm_mapping.id,
+                   timed_thinclient_to_vm_mapping.thinclient,
+                   timed_thinclient_to_vm_mapping.vm,
+                   timed_thinclient_to_vm_mapping.resolution,
+                   timed_thinclient_to_vm_mapping.start_date,
+                   timed_thinclient_to_vm_mapping.end_date,
+                   1 AS prio,
+                   timed_thinclient_to_vm_mapping.shutdown_vm
+      FROM timed_thinclient_to_vm_mapping
+      WHERE ((timed_thinclient_to_vm_mapping.start_date <= now())
+             AND (timed_thinclient_to_vm_mapping.end_date IS NULL)))
+   UNION SELECT timed_thinclient_to_vm_mapping.id,
+                timed_thinclient_to_vm_mapping.thinclient,
+                timed_thinclient_to_vm_mapping.vm,
+                timed_thinclient_to_vm_mapping.resolution,
+                timed_thinclient_to_vm_mapping.start_date,
+                timed_thinclient_to_vm_mapping.end_date,
+                2 AS prio,
+                timed_thinclient_to_vm_mapping.shutdown_vm
+   FROM timed_thinclient_to_vm_mapping
+   WHERE ((now() >= timed_thinclient_to_vm_mapping.start_date)
+          AND (now() <= timed_thinclient_to_vm_mapping.end_date))) f
+ORDER BY f.prio DESC,
+         f.start_date,
+         f.id;
 
```

Comment:
Much better than the oneliner from before, but again - use our SQL style guide :)